templates/test/ci/patches/azuremachinepool-vmextension.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureMachinePool
+metadata:
+  name: ${CLUSTER_NAME}-mp-0
+  namespace: default
+spec:
+  template:
+    vmExtensions:
+      - name: CustomScript
+        publisher: Microsoft.Azure.Extensions
+        version: '2.1'
+        protectedSettings:
+          commandToExecute: |
+            #!/bin/sh
+            echo "Updating packages ..."
```

Comment:
let's maybe make the echo statement less random and explain what this is for?

e.g. `echo "this script is a no-op used for extension testing purposes ..."`